packer/ansible/roles/localsource/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: install dev libraries for NPM installs
+  apt: pkg={{ item }} state=installed
+  with_items:
+    - libkrb5-dev
+    - unzip
+  sudo: yes
+
+- name: Install RackHD from local source
+  copy: src="../../{{ item }}" dest="{{ ansible_env.HOME }}/src/"
+  with_items:
+    - on-core
+    - on-tasks
+    - on-http
+    - on-tftp
+    - on-dhcp-proxy
+    - on-taskgraph
+    - on-syslog
+    - on-wss
+    - on-tools
+    - on-imagebuilder
+
+- name: Npm install Repos
+  npm: path="{{ ansible_env.HOME }}/src/{{ item }}"
+       production=yes
```

Comment:
Just a heads up that this may not get you what you expect.   For example, when you do an npm install in on-taskgraph, it is going to install the on-tasks and on-core modules from github rather than from local source unless you modify the package.json.

"dependencies": {
    "di": "git+https://github.com/RackHD/di.js.git",
    "dot-object": "^0.6.0",
    "lodash": "^3.10.1",
    "lru-cache": "^2.5.0",
    "node-uuid": "^1.4.2",
    "on-core": "git+https://github.com/RackHD/on-core.git",
    "on-tasks": "git+https://github.com/RackHD/on-tasks.git",
    "rx": "^2.3.22"
  },